client/src/cordova/apple/xcode/ios/Outline.xcodeproj/project.pbxproj

```diff
@@ -1156,7 +1156,7 @@
 				SWIFT_OBJC_BRIDGING_HEADER = "";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 5.0;
-				TARGETED_DEVICE_FAMILY = 2;
+				TARGETED_DEVICE_FAMILY = "6";
```

Comment:
6 is MacCatalyst "optimize for mac". 2 is "iPad, Mac Catalyst using “Scaled to Match iPad” Interface"

See https://developer.apple.com/documentation/xcode/build-settings-reference#Targeted-Device-Families

Reply:
You can specify multiple, like we do above:

```suggestion
				TARGETED_DEVICE_FAMILY = "2,6";
```

Not sure it's strictly necessary here, it'd be mostly for consistency.

Reply:
This is for the OutlineLauncher, which is Catalyst only. It doesn't apply to iPad. Same below (debug vs release)

Reply:
2 and 6 are both for Catalyst (2 just also covers iPad). It just depends on what interface is used. At one point we used the "Scaled to Match iPad" interface on Mac Catalyst, which corresponds to 2:

<img width="452" alt="Screenshot 2024-08-02 at 2 29 38 PM" src="https://github.com/user-attachments/assets/ba0d8c99-b5c2-46b2-8cfb-b76d6362da28">

Anyway not a big deal, I don't think it has a meaningful impact and we're getting rid of the launcher soon anyway.

Reply:
Aaahhh, I made the change, but forgot to save and push. I'll add it to the next PR